### DISCUSSION TITLE
Meeting data disappeared from the backend.

### DISCUSSION THREAD

**Discussion by FoxyCat111** [code4recovery/12-step-meeting-list#1191] [Q&A]
I'm using this plugin for a Canadian district.
The meeting schedule still presents on the website but the data seems to have disappeared from the back end.
Any idea what the problem could be?
Thanks for your attention to this.

**Comment by kiyote33** [comment]
Hi @FoxyCat111,
If I understand your situation correctly, your data has disappeared when you try to access your meeting information through the WordPress admin Meetings menu.  Is that correct? Could you relate what activities you were doing when this situation occurred?

**Comment by FoxyCat111** [comment]
Thank you – I’ve asked the other (non-technical) admin on the site your same inquiry. Will revert.

I just logged on to change the name of one of their meetings and there are no meetings posted in the plugin’s meeting section.

 

Yours sincerely

 

RICHARD SHUTE

+27825873920

 

 <https://www.planmatrixstrategic.com/> 

 

 <https://www.planmatrixstrategic.com/> PlanMatrix Strategic

 

 <https://planmatrix.web.za/> PlanMatrix Web

 

 <https://planmatrix.web.za/> #UploadYourEnterprise
